README.md

```diff
@@ -8,32 +8,29 @@ For VirtualMachines, it also allocates a mac address for the primary interface i
 
 ## Usage
 
-For test environment you can use the [development environment](#Develop)
+For test environment you can use the [development environment](#Develop).
 
 For Production deployment:
 
-Install any supported [Network Plumbing Working Group de-facto standard](https://github.com/k8snetworkplumbingwg/multi-net-spec) implementation.
-
-For example [Multus](https://github.com/intel/multus-cni).
-To deploy multus on a kubernetes cluster with flannel cni.
+* Install any supported [Network Plumbing Working Group de-facto standard](https://github.com/k8snetworkplumbingwg/multi-net-spec) implementation, for example [Multus](https://github.com/intel/multus-cni).
+  To deploy multus on a kubernetes cluster with flannel cni:
```

Comment:
We should consider use another CNI in this example (e.g., Calico), to prevent readers from encountering compatibility  issues with Flannel and Kubernetes (1.16 and above).
There is an opened issue about it in Flannel repo [1].

[1] https://github.com/coreos/flannel/issues/1245

Reply:
I hope this can wait for another PR. This PR is only about setting a random-like mac range.

Reply:
Sure no problem :)